HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -58,6 +58,13 @@ def customiseFor14833(process):
                 producer.includeME0 = cms.bool(False)
     return process
 
+def customiseFor16670(process):
```

Comment:
any reason a fillDescription does not work here? (with the nice feature of being desirable for the long term)